core/App/contexts/auth.tsx

```diff
@@ -15,6 +15,7 @@ export interface AuthContext {
   checkPIN: (pin: string) => Promise<boolean>
   convertToUseBiometrics: () => Promise<boolean>
   getWalletCredentials: () => Promise<WalletSecret | undefined>
+  wipeSavedWalletSecret: () => void
```

Comment:
better terminology for "wipe" like remove, clear or delete. I get wipe but maybe harder for non-english spears.